users_ldap_groups/__openerp__.py

```diff
@@ -30,13 +30,13 @@
 
```

Comment:
move this to README.rst, using https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst as template